modules/hcp-topology-spread-constraint.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-topology-spread-constraint_{context}"]
+= Spreading node pool VMs by using topologySpreadConstraint
+
+By default, KubeVirt virtual machines (VMs) created by a node pool are scheduled to any available node with the appropriate capacity to run the VM. A `topologySpreadConstraint` constraint spreads VMs of a node pool on multiple nodes. For example, if you are performing a live migration, you use `topologySpreadConstraint` with the descheduler so that VMs are continuously distributed on multiple nodes rather than a single node.
+
+.Prerequisites
+
+* You installed the {descheduler-operator}. For more information, see "Installing the descheduler".
+
+.Procedure
+
+* Modify the `KubeDescheduler` custom resource (CR) to use the `topologySpreadConstraint` constraint by running the following command:
```

Comment:
to use the `SoftTopologyAndDuplicates` profile to ensure that the soft `topologySpreadConstraint` constraint is going to be satisfied again when possible.